packages/ui/Skinnable/tests/Skinnable-specs.js

```diff
@@ -226,10 +225,13 @@ describe('Skinnable Specs', () => {
 
 		const SkinnableComponent = Skinnable(config, Component);
 
-		const subjectA = mount(<SkinnableComponent skinVariants="normal unicase" />);
-		const subjectB = mount(<SkinnableComponent skinVariants={['normal', 'unicase']} />);
+		const {rerender} = render(<SkinnableComponent data-testid="skinnableComponent1" skinVariants="normal unicase" />);
+		const component1ClassNames = screen.getByTestId('skinnableComponent1').className;
+
+		rerender(<SkinnableComponent data-testid="skinnableComponent2" skinVariants={['normal', 'unicase']} />);
```

Comment:
I think we should render instead of rerender.
```suggestion
		render(<SkinnableComponent data-testid="skinnableComponent2" skinVariants={['normal', 'unicase']} />);
```